packages/jellyfish-transaction/src/script/dftx/dftx_loans.ts

```diff
@@ -74,3 +76,29 @@ export class CDestroyLoanScheme extends ComposableBuffer<DestroyLoanScheme> {
     ]
   }
 }
+
+/**
+ * CreateVault DeFi Transaction
+ */
+export interface CreateVault {
+  ownerAddress: Script // --------------------| n = VarUInt{1-9 bytes}, + n bytes, Vault's owner address
+  schemeId: string // --------------------| c = VarUInt{1-9 bytes}, + c bytes UTF encoded string, Vault's loan scheme id
```

Comment:
alignment

Reply:
The code is not yet final and this is why I marked it as DRAFT. You probably can review my other **OPEN** issue.

Reply:
I'll text you tomorrow.